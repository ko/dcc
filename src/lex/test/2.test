int main(void)
{
    int a = 3;
    int b = 5;

    return 0;
}
