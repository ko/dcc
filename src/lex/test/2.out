[TInt,TIdentifier "main",TLeftParen,TVoid,TRightParen,TLeftBrace,TInt,TIdentifier "a",TAssign,TIntLiteral 3,TSemicolon,TInt,TIdentifier "b",TAssign,TIntLiteral 5,TSemicolon,TReturn,TIntLiteral 0,TSemicolon,TRightBrace]
